src/HealthGPS/analysis_module.h

```diff
@@ -46,6 +46,7 @@ class AnalysisModule final : public UpdatableModule {
     WeightModel weight_classifier_;
     DoubleAgeGenderTable residual_disability_weight_;
     std::vector<std::string> channels_;
+    std::unordered_map<std::string, int> channel_index_;
```

Comment:
I might not be completely understanding what these data structures do... But am I right in thinking `channel_index_` contains indexes into `channels_`? If so, could you just make `channels_` a `std::map<std::string, std::string>` (I'm guessing it needs to be ordered...) and drop `channels_index_`?